CMakeLists.txt

```diff
@@ -166,6 +166,12 @@ if (CONFIG_COMPILER_WARNINGS_AS_ERRORS)
   zephyr_link_libraries($<TARGET_PROPERTY:linker,warnings_as_errors>)
 endif()
 
+if (CONFIG_DEPRECATION_TEST)
```

Comment:
no space in cmake between if and bracket

Reply:
Needed to push so I fixed this now @nordicjm